doc/related_projects.rst

```diff
@@ -98,6 +102,11 @@ enhance the functionality of scikit-learn's estimators.
   custom matplotlib visualizers for scikit-learn estimators to support visual feature
   analysis, model selection, evaluation, and diagnostics.
 
+- `supertree <https://github.com/mljar/supertree>`_
```

Comment:
Could you move this entry under the first item since they are both about visualizing trees.

Reply:
I removed this line